gen/definitions/properties.yaml

```diff
@@ -185,15 +185,25 @@ pkiTP:
   default_values:
     parent_dn: "uni/userext/pkiext"
   overwrites:
+    cert_usage: "certificate_usage"
     cert_chain: "certificate_chain"
   resource_required:
     - "certChain"
+  ignore_properties_in_test:
+    certUsage: ""
```

Comment:
This would likely require a few changes in the generator and the resource_example templates 
 certUsage: "" -> certUsage: ["WebSvcOrAuth"]

Reply:
Yeah, I just tried it out. Looks like the generator only handles a single string instead of a list of strings. I'll look into it a little more.